dist/package/openSUSE-release-tools.spec

```diff
@@ -202,6 +202,7 @@ BuildArch:      noarch
 # TODO Update requirements.
 Requires:       osclib = %{version}
 Requires:       python-openqa_client
+Requires:       python2-pika
```

Comment:
Depending on what version package we use, we MIGHT need to require python-pika (in case the package we build is not py-singlespec)

Reply:
even if it isn't singlespeced we should rather have a python provide